doc/nrf/protocols/esb/index.rst

```diff
@@ -299,9 +299,10 @@ If you are sure that you do not require support for revision 2 chips, you may re
 Examples
 ========
 
-The |NCS| provides the following example application that shows how to use the ESB protocol:
+The |NCS| provides the following example application (as two samples) that shows how to use the ESB protocol:
```

Comment:
```suggestion
The |NCS| provides the following two samples that show how to use the ESB protocol:
```
There isn't really a single application, unless we mean the act of applying something, which can be confusing in this context.

Also, the Overview section of the documentation states it the other way around (sample with two applications).